app/packages/state/src/recoil/fieldVisibility.ts

```diff
@@ -0,0 +1,95 @@
+import { atom, selectorFamily } from "recoil";
+
+import { VALID_PRIMITIVE_TYPES } from "@fiftyone/utilities";
+
+import { expandPath, fields } from "./schema";
+import { State } from "./types";
+import { hiddenLabelIds } from "./selectors";
+
+export const modalFieldVisibility = atom<State.Filters>({
+  key: "modalFieldVisibility",
+  default: {},
+});
+
+export const fieldVisibility = atom<State.Filters>({
```

Comment:
the naming is very very similar to our Original Field Visibility feature. :)

Reply:
The sidebar features are working on the attribute level, so I just updated the names to attributeVisibility